File: AD-HOC/2414/main.c
#include <stdio.h>

int main() {
    int max = 0, num = 0;
    scanf("%d", &num);
    while (num > 0) {
        max = num > max ? num : max;
        scanf("%d", &num);
    }
    printf("%d\n", max);
    return 0;
}

File: AD-HOC/2414/Main.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int max = 0, num = 0;
        while ((num = scanner.nextInt()) > 0)
            max = num > max ? num : max;
        System.out.println(max);
        scanner.close();
    }
}